code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -581,6 +644,38 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 /datum/reagent/medicine/quickclot/overdose_crit_process(mob/living/L, metabolism)
 	L.apply_damages(0, 4*REM, 4*REM)
 
+
+/datum/reagent/medicine/quickclot_plus
+	name = "Quick Clot Plus"
+	description = "A chemical designed to completely repair internal bleeding. Significant toxic and shock side effects, do not use in combat."
+	color = "#CC00FF"
+	overdose_threshold = REAGENTS_OVERDOSE/30 //ONE unit.
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/25 //Two units
+	scannable = TRUE 
+	custom_metabolism = REAGENTS_METABOLISM * 0.25 //In body for 10 ticks
+
+/datum/reagent/medicine/quickclot_plus/on_mob_life(mob/living/L, metabolism)
+	L.apply_damage(10*REM, TOX) //50 toxin/unit
+	L.adjustHalLoss(300) //You do not get up while this is in you. At all.
```

Comment:
Stun effect instead, but again, balance concerns.

Reply:
Same as the last one.

Reply:
Stun changed to 4 seconds/tick.

Combat med balance concerns left unaddressed because it's not acquireable as of now, and because when it becomes acquireable, it'll be medic-only on a points basis.